src/string.rs

```diff
@@ -20,6 +20,13 @@ extern "C" {
     length: int,
   ) -> *const String;
 
+  fn v8__String__NewFromOneByte(
+    isolate: *mut Isolate,
+    data: *const char,
```

Comment:
```suggestion
    data: *const u8,
```
And `const uint8_t* data` in C++ land so you don't have to reinterpret_cast.